esp/templates/users/profiles/usercontact.html

```diff
@@ -69,11 +78,11 @@ <h4>Address:</h4>
         {% endif %}
     </div>
 </div>
-    
 {% endif %}
 
 {{ form.address_postal }}
 
+{% if form.phone_day or form.phone_cell %}
```

Comment:
If I remove phone_cell for student_reg, i can't even load the profile page anymore:
![image](https://user-images.githubusercontent.com/25392075/80314138-ae3ca480-87bd-11ea-8ea9-e589d335108b.png)